project.clj

```diff
@@ -14,6 +14,14 @@
                  [com.novemberain/langohr "5.0.0"]
                  [com.taoensso/nippy "2.14.0"]
                  [io.dropwizard.metrics5/metrics-core "5.0.0-rc2" :scope "compile"]
+                 [io.jaegertracing/jaeger-core "1.0.0"]
+                 [io.jaegertracing/jaeger-client "1.0.0"]
+                 [io.opentracing/opentracing-api "0.33.0"]
+                 [io.opentracing/opentracing-mock "0.33.0"]
```

Comment:
if this is only being used in tests, can we move it to tests profile?

Reply:
because we are using this in fixtures, it had to be added to the main deps.